src/app/project/project-details-tab/project-details-tab.component.ts

```diff
@@ -269,6 +275,26 @@ export class ProjectDetailsTabComponent implements OnInit, AfterViewInit, OnDest
 		}
 	}
 
+  /**
+   * The actions that should be run after all shape files are analyzed.
+   * 
+   * @param locationAdded A boolean that indicates whether or not at least one shapefile has been added to the map
+   */
+  private runShapeFilesAnalyzedActions = (locationAdded: boolean) => {
```

Comment:
This would describe what's happening inside the function so you could know at a glance when seeing it get executed:
```suggestion
  private calculateBoundsAndAddMarkers = (locationAdded: boolean) => {
```

Reply:
It also adds a layer to a feature group. So yes, I could give it a name like calculateBoundsAndAddMarkersOrAddLayerToFeatureGroup(), or I could title it based on the hook that we're using. I'll simplify it further to something like shapeFilesLoadedHook()